Deploy/Docker/MINIMAL/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN echo ${TAG}; \
     mkdir $HOME/work; \
     cd $HOME/work; \    
     git clone --recurse-submodules -b ${TAG} --single-branch --depth 1 https://github.com/qMRLab/qMRLab.git;\
-    chmod -R 777 $HOME/work/qMRLab; \
+    chmod -R 777 cd $HOME/work/qMRLab; \
```

Comment:
This `cd` seems out of place within a `chmod` command, I'm making a note to manually test the Dockerfile in case this needs to be fixed.